packages/main/src/themes/Input.css

```diff
@@ -21,10 +21,13 @@
 	word-spacing: normal;
 	text-align: start;
 	transition: var(--_ui5_input_transition);
-	background: var(--sapField_BackgroundStyle);
 	background-color: var(--_ui5_input_background_color);
 }
 
+:host(:not([disabled])) {
+	background: var(--sapField_BackgroundStyle);
+}
+
```

Comment:
Currently every input in every theme is missing its background-color because the 'background' property in your new selector is overriding it. You should add it in it:

```
:host(:not([disabled])) {
	background: var(--sapField_BackgroundStyle);
	background-color: var(--_ui5_input_background_color);
}
```

This way will work because the background-color is also set explicitly for [disabled] inputs too, later in the CSS file - line #79:

```
:host(:not([disabled])) {
	background: var(--sapField_BackgroundStyle);
	background-color: var(--_ui5_input_background_color);
}
```

Reply:
Thank you, I changed it.